configs/config.conf

```diff
@@ -10,5 +10,5 @@ KORYAVNIKS=(
 
 #DJVUVIEWER
 
-djvuviewer_script=~/gnu-koryavov/okular.sh
+djvuviewer_script=~/gnu-koryavov/atril.sh
```

Comment:
Please do not consider changing this value, because Okular seems to be the most widespread document viewer

Reply:
Okular is not present in all systems, for example in MacOS there are some problems with it. Atril definitely works on all Linux, MacOS and BSD systems. I changed it to Okular, but it's not very good idea.

Reply:
Up to you then. Since we have viewer autodetect maybe it's better to suggest installing Atril if there is no supported viewer installed

Reply:
I'll add README reference into ui for this situation in #39